assets/app.js

```diff
@@ -266,7 +266,25 @@ new Vue({
                 ]
             }
         },
-        translationUpdates: LogData.content
+        translationUpdates: LogData.content,
+        type_dict: {
```

Comment:
- handle as app property, move to `data`
- rename to `resourceDict` 
- we want to keep the resource relations accessible, therefor structure as
```
{
    baseProducts: {
      list: "baseProducts"
      name: "Base Products"
      selectableOptions: "optionsBaseProduct"
      translationRequested: "translatorBaseProducts"
    },
    components: {
        ....
    },
    ...
}